Mage.Client/src/main/java/org/mage/card/arcane/ModernCardRenderer.java

```diff
@@ -64,37 +65,6 @@ public class ModernCardRenderer extends CardRenderer {
 
     ///////////////////////////////////////////////////////////////////////////
     // Textures for modern frame cards
```

Comment:
miss comments move

Reply:
I've moved the comment to ModernCardResouceLoader.